specification/appcomplianceautomation/AppComplianceAutomation.Management/resources/EvidenceResource.tsp

```diff
@@ -16,6 +16,7 @@ namespace Microsoft.AppComplianceAutomation;
 /**
  * A class represent an AppComplianceAutomation evidence resource.
  */
+#suppress "@azure-tools/typespec-azure-core/no-private-usage" "Legacy DO NOT EVER COPY THIS"
```

Comment:
here too can we fix those use of armResourceInternal decorator or its because we still have limitation?